lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -40,11 +48,17 @@
   ingress:
     interface: ingress
 ```
+You _must_ register the IngressRequires class as part of the `__init__` method
+rather than, for instance, a config-changed event handler. This is because
+doing so won't get the current relation changed event, because it wasn't
+registered to handle the event (because it wasn't created in `__init__` when
```

Comment:
`since it wasn't created in ...` once more to avoid repetition of `because`

Reply:
This is the ingress library, directly pulled from charmhub, so we shouldn't be touching it here

Reply:
Erf, didn't realize it, then we should change this in the library itself